aimmo-game-worker/service.py

```diff
@@ -1,16 +1,35 @@
 #!/usr/bin/env python
+import os
 import logging
 import sys
 import json
+import traceback
+from contextlib import contextmanager
 
 import flask
+from flask_socketio import SocketIO
 
 from simulation.world_map import WorldMap
 from simulation.avatar_state import AvatarState
 
+from logger import SocketLogger
 from avatar import Avatar
 
 app = flask.Flask(__name__)
+socketio = SocketIO()
+logger = SocketLogger(socketio)
+
+
+@contextmanager
+def capture_output(new_stdout):
+    old_stdout = sys.stdout
+    sys.stdout = new_stdout
+    try:
+        yield
+    except Exception:
+        print(traceback.format_exc())
```

Comment:
Why do we not re-raise the exception?
Also, should this be called `capture_output_and_log_exceptions` if it will be doing both?